terraform/version.go

```diff
@@ -12,7 +12,7 @@ const Version = "0.9.6"
 // A pre-release marker for the version. If this is "" (empty string)
 // then it means that it is a final release. Otherwise, this is a pre-release
 // such as "dev" (in development), "beta", "rc1", etc.
-const VersionPrerelease = "dev"
+var VersionPrerelease string
```

Comment:
Should this still default to "dev" in the code, and explicitly set it to `""` for the release?